manageiq-operator/config/rbac/role_binding.yaml

```diff
@@ -1,11 +1,11 @@
-kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
 metadata:
+  name: manager-rolebinding
```

Comment:
manager or manageiq?

Reply:
It was autogenerated as manager, but I can change it

Reply:
I don;t have a preference...just thought it was a typo.